configure.ac

```diff
@@ -50,7 +50,8 @@ EKB_CXXFLAGS=" \
 -I${INCDIR}/hwpf/fapi2/include/error_info \
 -I${INCDIR}/hwpf/fapi2/include/plat \
 -I${INCDIR}/ekb/hwpf/fapi2/include \
--I${INCDIR}/ekb/chips/${CHIP}/procedures/hwp/ffdc"
+-I${INCDIR}/ekb/chips/${CHIP}/procedures/hwp/ffdc \
+-I${INCDIR}/ekb/chips/p10/procedures/hwp/lib"
```

Comment:
change to /ekb/chips/${CHIP}/procedures/hwp/lib

Reply:
done